R/Vietnam.R

```diff
@@ -0,0 +1,129 @@
+#' Vietnam Class for downloading, cleaning and processing
+#' notification data
+#'
+#' @description Information for downloading, cleaning
+#'  and processing covid-19 region data for Vietnam.
+#'
+#' @source \url{https://covid.ncsc.gov.vn}
+#' @export
+#' @concept dataset
+#' @family subnational
+#' @examples
+#' \dontrun{
+#' region <- Vietnam$new(verbose = TRUE, steps = TRUE, get = TRUE)
+#' region$return()
+#' }
+# nolint start
```

Comment:
`nolint` should be used for tighter frames of lines, because it helps us catch other problems with the code. If you remove this (and the `# nolint stop`) you get about a dozen lines warned from `lintr` (once you've also run `styler::style_file()`) and it would be better to mark these out specifically.